src/Symfony/Bundle/SecurityBundle/Tests/SecurityTest.php

```diff
@@ -201,6 +201,30 @@ public function testLoginWithoutAuthenticatorThrows()
         $security->login($user);
     }
 
+    public function testLoginWithoutRequestContext()
+    {
+        $requestStack = $this->createMock(RequestStack::class);
+        $user = $this->createMock(UserInterface::class);
+
+        $container = $this->createMock(ContainerInterface::class);
```

Comment:
I would also not mock the `ContainerInterface` but use the `ContainerBuilder` class instead

Reply:
@xabbuh done ! thanks

Reply:
All the tests in this class mock the `ContainerInterface`, I am confused why we would not do it for this case ?